mycelo/loadbot/bot.go

```diff
@@ -74,6 +81,14 @@ func runBot(ctx context.Context, acc env.Account, verbose bool, sleepTime time.D
 	transactor.FeeCurrency = &stableTokenAddress
 
 	for {
+		pendingLock.Lock()
```

Comment:
i would check golang atomic value, i thing they are exactly what you need
https://gobyexample.com/atomic-counters

Reply:
The first implementation did in fact use them. I had issue doing an atomic comparison though.